rsocket/ResumeManager.h

```diff
@@ -15,9 +15,17 @@ class IOBuf;
 
 namespace rsocket {
 
+struct StreamResumeInfo {
+  StreamType streamType;
+  Requester requester;
+  std::string streamToken{""};
```

Comment:
Nit: Don't need the `{""}` bit, default ctor of `std::string` is an empty string.